### PR TITLE
[velero] Sets default resource requests and limits

### LINKED ISSUE #231: Velero pod should have 512Mi for the memory limit
The limit for the Velero pod was bumped in release 1.5.3 from 256Mi to 512Mi due to crashes in the Azure plug-in with the smaller limit (https://github.com/vmware-tanzu/velero/issues/3234).  The Helm charts appears to be installing with the old limit (https://github.com/vmware-tanzu/helm-charts/issues/168)

### PR DESCRIPTION
**Pull request by jenting**
#### Special notes for your reviewer:

fixes #231

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the values.yaml or README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

**Review by zubron — APPROVED** [review]

File: charts/velero/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.5.3
 description: A Helm chart for velero
 name: velero
-version: 2.15.0
+version: 2.15.1
```
Comment (cpanato) [inline charts/velero/Chart.yaml:5]:
```suggestion
version: 2.16.0
```
because it is a feature addition and not a fix
besides that LGTM

**Review by cpanato — CHANGES_REQUESTED** [review]

File: charts/velero/Chart.yaml (hunk 1 above)
Comment (jenting) [reply charts/velero/Chart.yaml:5]: Since I was trying to fix an issue so I think it's a fix.

Comment (jenting) [reply charts/velero/Chart.yaml]: Never mind, agree with you.

**Comment by jenting** [comment]
@carlisia @zubron @cpanato PTAL 🙇

**Review by cpanato — APPROVED** [review]

**Comment by jenting** [comment]
@carlisia @zubron @cpanato PTAL

**Review by carlisia — APPROVED** [review]
Thank you 👍

**Review by zubron — APPROVED** [review]
Thanks!

**Pull request merged**